scene/resources/primitive_meshes.cpp

```diff
@@ -2151,3 +2156,798 @@ void RibbonTrailMesh::_bind_methods() {
+
+	ClassDB::bind_method(D_METHOD("set_text", "text"), &TextMesh::set_text);
+	ClassDB::bind_method(D_METHOD("get_text"), &TextMesh::get_text);
+
+	ClassDB::bind_method(D_METHOD("set_font", "font"), &TextMesh::set_font);
+	ClassDB::bind_method(D_METHOD("get_font"), &TextMesh::get_font);
+
+	ClassDB::bind_method(D_METHOD("set_font_size", "font_size"), &TextMesh::set_font_size);
+	ClassDB::bind_method(D_METHOD("get_font_size"), &TextMesh::get_font_size);
+
+	ClassDB::bind_method(D_METHOD("set_depth", "depth"), &TextMesh::set_depth);
+	ClassDB::bind_method(D_METHOD("get_depth"), &TextMesh::get_depth);
+
+	ClassDB::bind_method(D_METHOD("set_pixel_size", "pixel_size"), &TextMesh::set_pixel_size);
+	ClassDB::bind_method(D_METHOD("get_pixel_size"), &TextMesh::get_pixel_size);
+
+	ClassDB::bind_method(D_METHOD("set_curve_step", "curve_step"), &TextMesh::set_curve_step);
+	ClassDB::bind_method(D_METHOD("get_curve_step"), &TextMesh::get_curve_step);
+
+	ClassDB::bind_method(D_METHOD("set_text_direction", "direction"), &TextMesh::set_text_direction);
+	ClassDB::bind_method(D_METHOD("get_text_direction"), &TextMesh::get_text_direction);
+
+	ClassDB::bind_method(D_METHOD("set_opentype_feature", "tag", "value"), &TextMesh::set_opentype_feature);
+	ClassDB::bind_method(D_METHOD("get_opentype_feature", "tag"), &TextMesh::get_opentype_feature);
+	ClassDB::bind_method(D_METHOD("clear_opentype_features"), &TextMesh::clear_opentype_features);
+
+	ClassDB::bind_method(D_METHOD("set_language", "language"), &TextMesh::set_language);
+	ClassDB::bind_method(D_METHOD("get_language"), &TextMesh::get_language);
+
+	ClassDB::bind_method(D_METHOD("set_structured_text_bidi_override", "parser"), &TextMesh::set_structured_text_bidi_override);
+	ClassDB::bind_method(D_METHOD("get_structured_text_bidi_override"), &TextMesh::get_structured_text_bidi_override);
+
+	ClassDB::bind_method(D_METHOD("set_structured_text_bidi_override_options", "args"), &TextMesh::set_structured_text_bidi_override_options);
+	ClassDB::bind_method(D_METHOD("get_structured_text_bidi_override_options"), &TextMesh::get_structured_text_bidi_override_options);
+
+	ClassDB::bind_method(D_METHOD("set_uppercase", "enable"), &TextMesh::set_uppercase);
+	ClassDB::bind_method(D_METHOD("is_uppercase"), &TextMesh::is_uppercase);
+
+	ClassDB::bind_method(D_METHOD("_font_changed"), &TextMesh::_font_changed);
+	ClassDB::bind_method(D_METHOD("_request_update"), &TextMesh::_request_update);
+
+	ADD_GROUP("Text", "");
+	ADD_PROPERTY(PropertyInfo(Variant::STRING, "text"), "set_text", "get_text");
+	ADD_PROPERTY(PropertyInfo(Variant::OBJECT, "font", PROPERTY_HINT_RESOURCE_TYPE, "Font"), "set_font", "get_font");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "font_size", PROPERTY_HINT_RANGE, "1,127,1"), "set_font_size", "get_font_size");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "horizontal_alignment", PROPERTY_HINT_ENUM, "Left,Center,Right,Fill"), "set_horizontal_alignment", "get_horizontal_alignment");
```

Comment:
Since the documentation says only left, center, and right are supported, maybe we should remove the "Fill" option here?

Reply:
Fixed `pixel_size` not cleaning cache, added `width` for `Fill` alignment and updated docs for this and `Label3D` (both support `Fill` alignment).